conda_smithy/configure_feedstock.py

```diff
@@ -102,10 +105,10 @@ def render_circle(jinja_env, forge_config, forge_dir):
 
 @contextmanager
 def fudge_subdir(subdir):
-    orig = conda_build.metadata.cc.subdir
-    conda_build.metadata.cc.subdir = subdir
+    orig = cc.subdir
+    cc.subdir = subdir
     yield
-    conda_build.metadata.cc.subdir = orig
+    cc.subdir = orig
```

Comment:
Yep, I forgot.  Importing with "from" breaks things, because it essentially does this copying: http://stackoverflow.com/a/3536638/1170370